tests/ui/src/utilities/s3Helper.ts

```diff
@@ -1,28 +1,64 @@
+import { S3Client, PutObjectCommand, GetObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import { readFile } from 'fs/promises';
+import { createWriteStream } from 'fs';
+import { pipeline, Readable } from 'stream';
+import { promisify } from 'util';
 
 const bucket = `nhse-uec-cm-ui-test-bucket-${process.env.ENV}${process.env.WORKSPACE}`;
 
-export function addObject(sourceFilePath: string, targetFile: string) {
-  return require('child_process').execSync(
-    `aws s3api put-object --bucket ${bucket} --key ${targetFile} --body ${sourceFilePath}`
-)};
+const pipelineAsync = promisify(pipeline);
 
-export function getObject(sourceFile: string, targetFile?: string) {
+const client = new S3Client({ region: process.env.REGION });
+
+export async function addObject(sourceFilePath: string, targetFile: string) {
+  try {
+    const fileContent = await readFile(sourceFilePath);
+    const command = new PutObjectCommand({
+      Bucket: bucket,
+      Key: targetFile,
+      Body: fileContent,
+    });
+
+    const response = await client.send(command);
+    console.log('Success', response);
```

Comment:
For standard development having these logs might be usful but I don't think they are really necessary here as they are just utils for tests